TheWarWithin/Priorities/EvokerDevastation.simc

```diff
@@ -46,33 +50,36 @@ actions.aoe+=/firestorm,if=buff.snapfire.up
 # Acquire the buff
 actions.aoe+=/firestorm,if=talent.feed_the_flames
 # Grab Irid Red before Dragonrage without griefing extension
-actions.aoe+=/call_action_list,name=fb,if=talent.dragonrage&cooldown.dragonrage.up&talent.iridescence|empowering.fire_breath
```

Comment:
Need to have `|empowering.fire_breath` so that if FB is already charging, we can reach the entry that tells us how/when to finish the cast.